util/fibers/detail/scheduler.cc

```diff
@@ -521,24 +555,22 @@ auto Scheduler::RunWorkerFibersStepImpl() -> RunFiberResult {
     fi->SwitchTo();
 
     ProactorBase::UpdateMonotonicTime();
-    uint64_t now_ns = ProactorBase::GetMonotonicTimeNs();
+    ran_ns = ProactorBase::GetMonotonicTimeNs() - start_ns;
```

Comment:
so since we always preempt into NORMAL queue, this measurement is incorrect. imagine fiber F1 - background, fiber F2..F10 - are normal. F10 is being the dispatch fiiber. F1 runs, then switches to F2 and so on until it returns to dispatcher. Everything will be accounted for as `ran_ns` here.

Reply:
you should decide what you do when a BACKGROUND fiber preempts. Do we continiue with normal fibers if they exist? Do we switch back to the dispatcher fiber? Continue with other background fibers?